network-api/networkapi/wagtailpages/models.py

```diff
@@ -181,6 +181,23 @@ class OpportunityPage(MiniSiteNameSpace):
 
 @register_snippet
 class Petition(CTA):
+    # This field, if set, overrides the default SQS queue location so
+    # that the legacy location is used by the server, instead.
+    legacy_petition = models.BooleanField(
```

Comment:
How is this going to work, is the idea we go in and check this for all old petitions when we ship?

Reply:
could we make it a data migration?

Reply:
can do.